english/dig/dig.md

```diff
@@ -18,4 +18,6 @@ Do you:
 
 [Try to scare the bird?](bird/bird.md)
 
-[Take the marshmallow out of your pocket and offer it to the bird?](marshmallow_to_bird/marshmallow_to_bird.md)
\ No newline at end of file
+[Take the marshmallow out of your pocket and offer it to the bird?](marshmallow_to_bird/marshmallow_to_bird.md)
+
+[Reply the greeting by saying "How do you do?" to the bird.](reply-greeting/reply-greeting.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.